github-pages.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency("mercenary", "~> 0.3")
   s.add_dependency("nokogiri", ">= 1.13.6", "< 2.0")
-  s.add_dependency("terminal-table", "~> 1.4")
+  s.add_dependency("terminal-table", "~> 3.0")
```

Comment:
We might prefer something like >= 1.4, <4 for now unless our use is broken between these versions?

Reply:
Will update.